modin/config/envvars.py

```diff
@@ -94,7 +94,11 @@ class Backend(EnvironmentVariable, type=str):
 
     varname = "MODIN_BACKEND"
     default = "Pandas"
+<<<<<<< HEAD
```

Comment:
Merge conflict

Reply:
Hey there Luis, do you know if refreshing helps? I pushed a commit pretty soon after I'd realized there were more merge conflicts that VSCode wasn't showing. I just want to make sure that those conflicts were taken out.... on my end, it says Outdated. So please let me know if it's not there.

Reply:
I think there are a couple one that were taken out! There is one in modin/engines/ray/cudf_on_ray/series/__init__.py . Also, the files in modin/engines/ray/cudf_on_ray/readers/* must be deleted.